src/main/java/org/kainos/ea/resources/JobRoleController.java

```diff
@@ -71,5 +73,28 @@ public Response getJobRoleById(@HeaderParam("Authorization") String token, @Path
         }
     }
 
-
+    @DELETE
+    @Path("/job-roles/{id}")
+    @Produces(MediaType.APPLICATION_JSON)
+    public Response deleteJobRoleById(@HeaderParam("Authorization") String token, @PathParam("id") int id) {
+        try {
+            String permission = "Admin";
+            authService.isValidToken(token, permission);
+            return Response
+                    .status(Response.Status.ACCEPTED)
```

Comment:
Is a 202 the best response code for this? Could a `204` be used better?

Reply:
Wasn't too sure. -> have updated to 204, however 204 makes sense...